src/Microsoft.AspNetCore.ResponseCompression/ResponseCompressionExtensions.cs

```diff
@@ -13,22 +14,53 @@ namespace Microsoft.AspNetCore.Builder
     public static class ResponseCompressionExtensions
     {
         /// <summary>
-        /// Allows to compress HTTP Responses.
+        /// Add response compression services and enable compression for responses with the given MIME types.
+        /// </summary>
+        /// <param name="collection">The service collection.</param>
+        /// <param name="mimeTypes">Response Content-Type MIME types to enable compression for.</param>
+        /// <returns></returns>
+        public static IServiceCollection AddResponseCompression(this IServiceCollection collection, params string[] mimeTypes)
+        {
+            return collection.AddResponseCompression(options =>
+            {
+                options.MimeTypes = mimeTypes;
+            });
+        }
+
+        /// <summary>
+        /// Add response compression services and configure the related options.
+        /// </summary>
+        /// <param name="collection"></param>
```

Comment:
nit: we usually call it `services` instead of `collection`? Also, elaborate on the params and return types of the xml docs?